src/constants/notepadActions.js

```diff
@@ -0,0 +1,12 @@
+export const NOTEPAD_ACTIONS = {
+  BOLD: 'BOLD',
+  bold: 'bold',
```

Comment:
I'm not sure about it, but there is no need to have two constants with the same value (nevermind that one is Uppercase and the other in lower case).

Try leaving only the uppercasse constants and let me know if it doesn't work!